### ISSUE TITLE
Document `standalone_package` target

### ISSUE THREAD

**Issue by i-ky** [Ericsson/codechecker#3299]
**Is your feature request related to a problem? Please describe.**
I'm was trying to put CodeChecker into Docker container and I ended up checking out code from GitHub and making `standalone_package` target instead of `package`, because it greatly simplifies container usage. It took me a while to find this solution, I can't recall exactly, but I believe I had to look into Makefile.

**Describe the solution you would like**
Would be nice to document `standalone_package` target somewhere [here](https://github.com/Ericsson/codechecker#linux). I think it is quite useful for someone who wants to install CodeChecker from source without any intentions to develop it.

**Describe alternatives you have considered**
Unfortunately, using Snap packages in Docker container is a bit tricky.

**Additional context**
As far as I know, WSL is not compatible with snap as well.

**Comment by csordasmarton** [comment]
@i-ky Our plan is to create a pypi package from the CodeChecker too which can be used in all distributions (Linux, OSX, Windows). I already created a patch for it (#3251) and if it is merged, I will try to create an official pypi package from it which can be installed easily by using `pip install codechecker` command. I think this way it will be easier to install CodeChecker in Docker environment too.

**Comment by i-ky** [comment]
Yes, that would be a nice alternative to snap for both Docker and WSL.

**Comment by bruntib** [comment]
CodeChecker is deployed to PyPI
